tests/test_RestartSerialization.cpp

```diff
@@ -134,6 +134,64 @@ BOOST_AUTO_TEST_CASE(EclGenericProblem)
     BOOST_CHECK_MESSAGE(data_out == data_in, "Deserialized EclGenericProblem differ");
 }
 
+namespace {
+
+struct AquiferFixture {
+    AquiferFixture() {
+        using TT = Opm::Properties::TTag::EbosTypeTag;
+        const char* argv[] = {
+            "test_RestartSerialization",
+            "--ecl-deck-file-name=GLIFT1.DATA"
+        };
+        Opm::setupParameters_<TT>(2, argv, /*registerParams=*/true);
+        Opm::EclGenericVanguard::setCommunication(std::make_unique<Opm::Parallel::Communication>());
```

Comment:
Does `Parallel::Communication` work in sequential mode?  If not, we should probably protect this with `HAVE_MPI` or similar.

Reply:
It's handled by a dummy in dune so it works fine (mostly). The issue we have encountered sometimes is that the dummy communication is not castable to Dune::No_Comm, while the MPI communicator is castable to MPI_Comm. In this case this is not relevant so no ifdef needed.